test/testdrive/fetch-tail-as-of.td

```diff
@@ -26,6 +26,13 @@ out of range integral type conversion attempted
 
 > COMMIT
 
+# Since FETCH isn't idempotent and can't be correctly retried by testdrive
+# (although testdrive will try), wait until SELECT fails with this error
```

Comment:
Testdrive has been modified already to not retry FETCH

Reply:
Hrm when did that happen? In my testing it did retry.